docs/source/evaluation_scripts.md

```diff
@@ -80,3 +82,67 @@ Let's break down what is happening in the above code snippet.
 1. `output` should contain a key named `result`, which is a list containing entries per dataset split that is available for the challenge phase in consideration (in the function definition of `evaluate()` shown above, the argument: `phase_codename` will receive the _codename_ for the challenge phase against which the submission was made).
 2. Each entry in the list should be a dict that has a key with the corresponding dataset split codename (`train_split` and `test_split` for this example).
 3. Each of these dataset split dict contains various keys (`Metric1`, `Metric2`, `Metric3`, `Total` in this example), which are then displayed as columns in the leaderboard.
+
+### Writing a remote evaluation script
+
+Just like a regular evaluation script, a remote evaluation script allows you to customize evaluation for your remote challenge. The starter template for remote challenge evaluation can be found [here](https://github.com/Cloud-CV/EvalAI-Starters/blob/master/remote_challenge_evaluation/evaluation_script_starter.py).
+
+There are a few things that you need to configure:
+
+1. **Set variables in the `__init__` method**: There are few variables that you need to set according to your challenge. The `auth_token` is fetched from the profile and `evalai_api_server` is set according to the server. The `challenge_pk` and `queue_name` can be fetched from the `Manage` tab of your challenge once it is up on  the server.
+
+    Changing the `save_dir` is optional and up to your preference.
+
+    ```python
+    self.auth_token = ""  # Go to EvalAI UI to fetch your auth token
+    self.evalai_api_server = ""  # For staging server, use -- https://staging.eval.ai; For production server, use -- https://eval.ai
+    self.queue_name = ""  # Check Manage Tab of challenge for queue name
+    self.challenge_pk = ""  # Check Manage Tab of challenge for challenge PK
+
+    self.save_dir = "./" # Location where submissions are downloaded
+    ```
+
+2. **Edit the `evaluate` method**: Like a regular evaluation, the `evaluate` method in `RemoteEvaluationUtil` is similar. It includes the three arguments - `user_annotation_file`, `phase_codename`, `test_annotation_file`.
```

Comment:
Change to

```
Write `evaluate` method:
```
Refer to [this documentation page](https://evalai.readthedocs.io/en/latest/evaluation_scripts.html) and just copy over the content from 
`Evaluation scripts are required to have an evaluate() function. This is the main function, which is used by workers to evaluate the submission messages.` line and make relevant edits to the  method signature, variables, etc.